easybuild/tools/docs.py

```diff
@@ -564,7 +564,14 @@ def avail_toolchain_opts(name, output_format=FORMAT_TXT):
         raise EasyBuildError("Couldn't find toolchain: '%s'. To see available toolchains, use --list-toolchains" % name)
     tc = tc_class(version='1.0') # version doesn't matter here, but needs to be defined
 
-    options = [tc.COMPILER_SHARED_OPTS, tc.COMPILER_UNIQUE_OPTS, tc.MPI_SHARED_OPTS, tc.MPI_UNIQUE_OPTS]
+    mpishared = []
+    mpiunique = []
```

Comment:
this won't work, since you use these values in `.update(..)`, so they should be `{}` instead?